core/java/iesi-core/src/main/java/io/metadew/iesi/script/action/http/HttpExecuteRequest.java

```diff
@@ -132,6 +114,99 @@ protected boolean executeAction() throws NoSuchAlgorithmException, IOException,
         return true;
     }
 
+    @Override
+    protected String getKeyword() {
+        return ACTION_TYPE;
+    }
+
+    private <T extends HttpParameter> List<T> combineParameters(List<T> componentLevelParameters, List<T> actionLevelParameters) {
```

Comment:
I don't think you need the T extends HttpParameter, can you just try the superclass HttpParameter

Reply:
The code doesn't comply when trying with the superClass. It says : 
required : List<HttpParameter>
provided: List<HttpHeader>

It's why I used generic

Reply:
you mean the return type?

Reply:
The parameters, when I defined the parameters as List < HttpParameter > it doesn't complain because I provide list HttpHeader instead of list of HttpParameter.